lldb/test/API/macosx/sme-registers/main.c

```diff
@@ -0,0 +1,113 @@
+///  BUILT with
```

Comment:
Nit: `s/built/build/`. But also this is covered by the makefile so maybe something like "Requires -mcpu=apple-m4" would be more to the point.